lms/djangoapps/teams/views.py

```diff
@@ -1384,7 +1382,16 @@ def post(self, request, **_kwargs):
         Process uploaded CSV to modify team memberships for given course run.
         """
         self.check_access()
-        return HttpResponse(status=status.HTTP_501_NOT_IMPLEMENTED)
+        inputfile_handle = request.FILES['csv']
+        team_import_manager = TeamMembershipImportManager(self.course)
+        team_import_manager.set_team_membership_from_csv(inputfile_handle)
+        if team_import_manager.import_succeeded is True:
+            return Response(team_import_manager.number_of_record_added.__str__(), status=status.HTTP_201_CREATED)
+        else:
+            return Response(
+                build_api_error(':'.join([e for e in team_import_manager.validation_errors])),
```

Comment:
Is this the most helpful format? I'd vote for something either more human-readable or machine parseable so we can surface it better.